src/app/portlets/content-types/fields/content-types-fields-list/content-types-fields-list.component.ts

```diff
@@ -15,8 +15,25 @@ import { FieldUtil } from '../util/field-util';
     templateUrl: './content-types-fields-list.component.html'
 })
 export class ContentTypesFieldsListComponent implements OnInit {
+    @Input() contentBaseType: string;
+
     fieldTypes: { clazz: string; name: string }[];
 
+    private dotFormFields = [
```

Comment:
instead of `clazz` how about using `baseType` attr?

Reply:
nvm, seems like the endpoint `v1/fieldTypes` does not returns the `baseType` attr